src/ng/filter/filters.js

```diff
@@ -358,10 +358,11 @@ function dateGetter(name, size, offset, trim) {
   };
 }
 
-function dateStrGetter(name, shortForm) {
+function dateStrGetter(name, shortForm, formatProp) {
   return function(date, formats) {
     var value = date['get' + name]();
-    var get = uppercase(shortForm ? ('SHORT' + name) : name);
+    var propName = formatProp || name;
+    var get = uppercase(shortForm ? ('SHORT' + propName) : propName);
```

Comment:
Good point ! (It was late :stuck_out_tongue:)
I went with the boolean (but it's easy to switch to a string prefix if needed).